src/plugins/internal/pluginsservice.cpp

```diff
@@ -25,18 +25,55 @@
 #include "view/pluginview.h"
 #include "pluginserrors.h"
 
+#include "pluginsuiactions.h"
+
 #include "log.h"
 
 using namespace mu::plugins;
 using namespace mu::framework;
 using namespace mu::async;
+using namespace mu::shortcuts;
+
+using ConfiguredPlugin = IPluginsConfiguration::ConfiguredPlugin;
+using ConfiguredPluginHash = IPluginsConfiguration::ConfiguredPluginHash;
+
+static const std::string PLUGINS_CONTEXT = "plugins";
+
+std::string shortcutsToString(const std::vector<std::string>& shortcuts)
```

Comment:
You can find

static std::string sequencesToString(const std::vector<std::string>& seqs)

in shortcutstypes.h

Better to use it instead of this function